Mathlib/Algebra/Ring/CentroidHom.lean

```diff
@@ -131,8 +133,14 @@ theorem coe_toAddMonoidHom_injective : Injective ((↑) : CentroidHom α → α
     this
 #align centroid_hom.coe_to_add_monoid_hom_injective CentroidHom.coe_toAddMonoidHom_injective
 
+lemma map_mul_left (f : CentroidHom α) (a b : α) : f (a * b) = a * f b :=
```

Comment:
why do you even need this lemma? `map_mul_left f` should Just Work(TM), and it's already `export`ed to the root namespace.
```suggestion
protected lemma map_mul_left (f : CentroidHom α) (a b : α) : f (a * b) = a * f b :=
```

Reply:
This lemma is no longer needed, so I removed it.